integration/shareddata/composites.go

```diff
@@ -28,10 +32,14 @@ var Composites = &Values[string]{
 		"document-empty":             bson.D{},
 
 		"array":               bson.A{int32(42)},
+		"array-two":           bson.A{42.42, math.NaN()},
```

Comment:
```suggestion
		"array-two":           bson.A{42.13, math.NaN()},
```

We use 42.13 in scalars; let's use the same value as it probably will trigger more cases